components/form-builder/panel/PanelActions.tsx

```diff
@@ -24,6 +24,26 @@ const PanelButton = styled(Button)`
   border: 1px solid transparent;
   padding: 20px 5px;
   transition: background 0.1s ease, border 0.1s linear;
+
+  &.disabled {
+    color: #ccc;
+    cursor: not-allowed;
+
+    svg {
+      fill: #ccc;
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+      color: #ccc;
+      background-color: #eee;
+
+      svg {
+        fill: #ccc;
+      }
+    }
+  }
```

Comment:
There is a very subtle background colour here that I don't think we want. Also the focus ring, we should turf it.

It's a `box-shadow: none` to get rid of the blue focus ring.

Also, it's not going to pass a11y (which, you could argue, is not a huge problem if you make the element non-focusable).

![Screen Recording 2022-09-15 at 16 50 23](https://user-images.githubusercontent.com/2454380/190508385-b5c893b1-b2be-42b2-92eb-eb88d48d0ffc.gif)